tests/conftest.py

```diff
@@ -0,0 +1,76 @@
+import asyncio
+import random
+import string
+
+import asyncpg
+import pytest
+
+from pathlib import Path
+
+from swoop.api.config import Settings, get_settings
+
+
+@pytest.fixture(scope="session")
+def dbschema(pytestconfig) -> Path:
+    return pytestconfig.rootpath.joinpath("db", "schema.sql")
+
+
+@pytest.fixture(scope="session")
+def settings() -> Settings:
+    return get_settings()
+
+
+async def async_create_database(db_name: str, db_connection_string: str) -> None:
+    conn = None
+    try:
+        conn = await asyncpg.connect(db_connection_string)
+        await conn.execute(f'CREATE DATABASE "{db_name}";')
+    finally:
+        if conn:
+            await conn.close()
```

Comment:
This pattern probably should be extracted into a context manager:

```
from contextlib import asynccontextmanager

@asynccontextmanager
async def get_db_connection(db_connection_string: str):
    conn = None
    try:
        conn = await asyncpg.connect(db_connection_string)
        yield conn
    finally:
        if conn:
            await conn.close()
```

Which would make this function body simply:

```suggestion
    with get_db_connection(db_connection_string) as conn:
        await conn.execute(f'CREATE DATABASE "{db_name}";')
```

Reply:
This `get_db_connection()` function will likely also be useful for loading data fixtures into a test database.